AutomatedTesting/Gem/PythonTests/atom_renderer/atom_hydra_scripts/hydra_AtomEditorComponents_LightComponent.py

```diff
@@ -0,0 +1,225 @@
+    ("Controller|Configuration|Shadows|Shadowmap size", 2),  # 1024
+    ("Controller|Configuration|Shadows|Shadowmap size", 3),  # 2048
+    ("Controller|Configuration|Shadows|Shadow filter method", 1),  # PCF
+    ("Controller|Configuration|Shadows|Filtering sample count", 4.0),
+    ("Controller|Configuration|Shadows|Filtering sample count", 64.0),
+    ("Controller|Configuration|Shadows|PCF method", 0),  # Bicubic
+    ("Controller|Configuration|Shadows|PCF method", 1),  # Boundary search
+    ("Controller|Configuration|Shadows|Shadow filter method", 2),  # ECM
+    ("Controller|Configuration|Shadows|ESM exponent", 50),
+    ("Controller|Configuration|Shadows|ESM exponent", 5000),
+    ("Controller|Configuration|Shadows|Shadow filter method", 3),  # ESM+PCF
+]
+QUAD_LIGHT_PROPERTIES = [
+    ("Controller|Configuration|Both directions", True),
+    ("Controller|Configuration|Fast approximation", True),
+]
+SIMPLE_POINT_LIGHT_PROPERTIES = [
+    ("Controller|Configuration|Attenuation radius|Mode", 0),
+    ("Controller|Configuration|Attenuation radius|Radius", 100.0),
+]
+SIMPLE_SPOT_LIGHT_PROPERTIES = [
+    ("Controller|Configuration|Shutters|Inner angle", 45.0),
+    ("Controller|Configuration|Shutters|Outer angle", 90.0),
+]
+
+
+def verify_required_component_property_value(entity_name, component, property_path, expected_property_value):
+    """
+    Compares the property value of component against the expected_property_value.
+    :param entity_name: name of the entity to use (for test verification purposes).
+    :param component: component to check on a given entity for its current property value.
+    :param property_path: the path to the property inside the component.
+    :param expected_property_value: The value expected from the value inside property_path.
+    :return: None, but prints to general.log() which the test uses to verify against.
+    """
+    property_value = editor.EditorComponentAPIBus(
+        bus.Broadcast, "GetComponentProperty", component, property_path).GetValue()
+    general.log(f"{entity_name}_test: Property value is {property_value} "
+                f"which matches {expected_property_value}")
+
+
+def run():
+    """
+    Test Case - Light Component
+    1. Creates a "light_entity" Entity and attaches a "Light" component to it.
+    2. Updates the Light component to each light type option from the LIGHT_TYPES constant.
+    3. The test will check the Editor log to ensure each light type was selected.
+    4. Prints the string "Light component test (non-GPU) completed" after completion.
+
+    Tests will fail immediately if any of these log lines are found:
+    1. Trace::Assert
+    2. Trace::Error
+    3. Traceback (most recent call last):
+
+    :return: None
+    """
+    # Create a "light_entity" entity with "Light" component.
+    light_entity_name = "light_entity"
+    light_component = "Light"
+    light_entity = hydra.Entity(f"{light_entity_name}")
```

Comment:
f-string not needed, but I don't have a problem leaving it as is or changing it, either way.
hydra.Entity(light_entity_name)

Reply:
ill update it